crates/topos-config/src/tce/broadcast.rs

```diff
@@ -0,0 +1,23 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct ReliableBroadcastParams {
+    /// Echo threshold
```

Comment:
I have read the PR but it is not clear to me what these threshold actually are. Is this how many messages of a given type must be received (or sent?) to be considered sent? Or what are they?

Reply:
That the thresholds of the reliable broadcast protocol.